fdbserver/storageserver.actor.cpp

```diff
@@ -1156,7 +1156,6 @@ ACTOR Future<GetKeyValuesReply> readRange( StorageServer* data, Version version,
 
 			for (auto i = &result.data[prevSize]; i != result.data.end(); i++) {
 				*pLimitBytes -= sizeof(KeyValueRef) + i->expectedSize();
-				readSize += sizeof(KeyValueRef) + i->expectedSize();
```

Comment:
I'm trying to understand where the `readSize` is counted for `rangeRead` after this line is removed. 
A wild guess is that `findKey ` counts the `readSize`. Is that correct?

Reply:
Yes. `readRange()` no longer reports read costs.

Reply:
So if a user only issues small range read at high frequency, the sampling will not detect it. Is that correct?

Reply:
It will. Small range reads will still be logged in `getValues()`.